finality-aleph/src/network/service.rs

```diff
@@ -843,6 +896,312 @@ mod tests {
         test_data.cleanup().await
     }
 
+    #[tokio::test]
+    async fn test_generic_data_command_send_to() {
```

Comment:
Do you think those new tests can be grouped in pairs with existing ones to extract some base methods and reuse them in those two flavors? E.g. extract this test and `test_validator_data_command_send_to` to come common either parametrized or generic base method

Reply:
I can do it, but I don't think that such version is better. When current tests fail you already get a few functions to check, I don't want to add more layers